spring-cloud-context/src/main/java/org/springframework/cloud/context/restart/RestartEndpoint.java

```diff
@@ -114,7 +114,7 @@ public ResumeEndpoint getResumeEndpoint() {
 		return new ResumeEndpoint();
 	}
 
-	@ConfigurationProperties("endpoints")
+	@ConfigurationProperties("endpoints.pause")
```

Comment:
I know, but this IMO not the best solution, since I would like to also be able to disable the pause endpoint independently, on top of globally. The same goes for the resume endpoint.

The global endpoints.enable logs is now handled byt @ConditionalOnProperty in RefreshEndpointAutoConfiguration.